.hlint.yaml

```diff
@@ -18,6 +18,9 @@
 - ignore: {name: "Use <>"}
 - ignore: {name: "Use fewer imports"}
 - ignore: {name: "Use camelCase"}
+- ignore: {name: "Use <$>"}
+- ignore: {name: "Redundant bang pattern"}
```

Comment:
Where are we using Redundant bang pattern?

Reply:
```haskell